### PR TITLE
Developing Load Screen: Graph Info loads, graph data does not

### PR DESCRIPTION
**Pull request by picaq**
Trello Ticket: [load data from db as modal component](https://trello.com/c/bFOTwhUe/25-load-data-from-the-db-as-a-modal-component)
heroku: https://prerequisite.herokuapp.com/

- Info loads into Load component by clicking Load button
  - graph data does not ( using id params )
    - there is no UI button to load graph data yet ( working from back to front )

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 FROM node:lts-alpine as react
 ARG NODE_ENV=production
+# ARG REACT_APP_NASA_API_KEY
```
Comment (m3ia) [inline Dockerfile]: Nice use of the NASA external API

File: app/src/App.js [hunk 2]
```diff
@@ -244,6 +230,45 @@ const App = () => {
...
   };
 
+  // loading list from db
+  const [graphInfo, setGraphInfo] = React.useState([]);
+
+  const getGraphInfo = async () => {
+    try {
+      setGraphInfo(await apiClient.getGraphs());
+      console.log(graphInfo);
+    } catch (error) {
```
Comment (m3ia) [inline app/src/App.js:238]: Creative way to catch errors!

File: app/src/components/LoadScreen.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        <thead>
+          <tr>
+            <th>id</th>
+            <th>Graph Name</th>
+            {/* <th>User</th> */}
+            {/* <th>Time Saved</th> */}
+          </tr>
+        </thead>
+        <tbody>
+          {graphInfo.map((info) => (
```
Comment (m3ia) [inline app/src/components/LoadScreen.js:36]: This seems to work fine although I'm confused as to what info is populating. Seems like a WIP?

**Review by m3ia — APPROVED** [review]
It's cool to see your app working the way you initially intended at this point! Seems ready to merge.

File: app/src/components/LoadScreen.js (hunk 3 above)
Comment (picaq) [reply app/src/components/LoadScreen.js:36]: Yes, I intend these to link to an id param to load data from a single row, but that doesn’t work yet

**Pull request merged**